preprocessing/flex/tier_registry.js

```diff
@@ -98,11 +98,11 @@ class tierRegistry {
   getTierName(lang, type) {
     /*
     // English UI text:
-      return decodeLang(lang) + " " + decodeType(type);
+      return decodeLang(lang) + " in " + decodeType(type);
```

Comment:
"in" is incorrect - e.g. it should be "A'ingae word" not "A'ingae in word". English vs Spanish should be different here.